macros/lookback.sql

```diff
@@ -0,0 +1,9 @@
+{% macro lookback(from_date, datepart='day', interval=7, default_start_date='2010-01-01') %}
```

Comment:
I realize calling this value `default_start_date` is a bit misleading. The `2010-01-01` is only meant as a super cautious failsafe in case the of null values in an incremental run, which I imagine would rarely occur. I'll rename this. The actual start date of events being brought in can be set by variable `date_range_start`. Thanks for the callout!